testcases/PowerNVDump.py

```diff
@@ -663,14 +665,30 @@ class KernelCrash_KdumpSSH(PowerNVDump):
     # Need to pass --dump-server-ip and --dump-server-pw in command line.
     # Needs passwordless authentication setup between test machine and ssh server.
 
+    def setUp(self):
+        super(KernelCrash_KdumpSSH, self).setUp()
+
+        conf = OpTestConfiguration.conf
+        self.dump_location = self.dump_server_ip
+        self.dump_server_private_ip = conf.args.dump_server_private_ip if 'dump_server_private_ip' in conf.args else None
+        if self.dump_server_private_ip:
+            self.interface = conf.args.interface
+            self.interface_ip = conf.args.interface_ip
+            self.netmask = conf.args.netmask
+            self.dump_location = self.dump_server_private_ip
+            self.util.configure_host_ip(self.interface, self.interface_ip, self.netmask, self.cv_HOST)
```

Comment:
is this configuring network by using `ip commands `  ? if yes .. than the network will not be persisitent